packages/yoroi-extension/app/stores/ada/AdaTransactionsStore.js

```diff
@@ -11,6 +11,11 @@ import type { StoresMap } from '../index';
 import WalletTransaction from '../../domain/WalletTransaction';
 import CardanoShelleyTransaction from '../../domain/CardanoShelleyTransaction';
 import type { RemoteTokenInfo } from '../../api/ada/lib/state-fetch/types';
+import type { NetworkRow } from '../../api/ada/lib/storage/database/primitives/tables';
+
+function createTokenLocalStorageKey(tokenId: string, network: $ReadOnly<NetworkRow>): string {
+  return `token-metadata-${network.NetworkName}-${tokenId}`;
```

Comment:
Maybe better to use `NetworkId` instead of `NetworkName`?

Reply:
Done 👌